tox.ini

```diff
@@ -21,7 +21,7 @@ commands =
 # To execute, run `tox -e pytest`
```

Comment:
i'm curious what the changes in this file are

Reply:
increased verbosity on a couple of commands, added a couple flake8 rules to ignore, and removed a couple of commands from the linting command (docstring style and mypy)